lib/sle2docker/prebuilt_image.rb

```diff
@@ -103,7 +103,7 @@ def compute_repository_and_tag
       match = regexp.match(@image_name)
       if match.nil?
         fail DockerTagError,
-             "Cannot calculate the Docker tag for #{@image_name}"
+             "Docker image #{@image_name} not found"
```

Comment:
I'm actually somewhat confused on this one. I understand that it's parsed through the regexp however if they didn't type it in the correct format then it isn't going to be found. This is true for any typo or failure to type things in correctly. I'm not sure how you would expect step one of your suggestion when we already failed to check if it existed the first time. Truly the image that they entered wasn't found and at that point they could review image names if they wanted to compare and make sure they typed it in correctly. 

Pardon if I'm not understanding, feel free to educate me on this. :) Thanks